docker/build.sh

```diff
@@ -184,8 +184,10 @@ function docker_cmd()
             DOCKER_TAG_TESTS=${GIT_SHA:0:12}
         fi
 
-        # builds the juptyer notebooks docker image that goes on dataproc clusters
-        bash ./jupyter-docker/build.sh build "${NOTEBOOK_REPO}" "${DOCKER_TAG}"
+        # builds the juptyer notebooks and rstudio docker images that go on dataproc clusters
+        bash ./docker/build-tool.sh -i jupyter -r "${NOTEBOOK_REPO}" -t "${DOCKER_TAG}"
```

Comment:
now that the number of tools is growing, should we start thinking of a way to decouple these builds from the leo app build?

Reply:
yeah I think that's a good idea. Maybe out of scope of this PR though.

Reply:
agreed that it's out of scope :+1: